apps/typeform/lambda/forms-handler.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fetchForms = require('./fetch-forms');
+
+module.exports = async (method, path, token, { fetch }) => {
+  if (method !== 'GET') {
+    return {
+      status: 405,
+      body: { message: 'Method not allowed.' }
+    };
+  }
+
+  try {
+    return {
+      status: 200,
+      body: { forms: await fetchForms(method, path, token, { fetch }) }
+    };
+  } catch (err) {
+    return {
+      status: 400,
```

Comment:
I'm wondering if we should actually pass through the status that was given to us by the fetch on line 16. Otherwise, all requests that fail will just show up as a 400 in our logs which won't be helpful for us.

Reply:
Ya we should probably do that